packages/composite-checkout/src/components/checkout-payment-methods.js

```diff
@@ -77,16 +77,14 @@ export default function CheckoutPaymentMethods( { summary, isComplete, className
 				{ paymentMethods.map( ( method ) => (
 					<CheckoutErrorBoundary
 						key={ method.id }
-						errorMessage={
-							localize( 'There was a problem with the payment method:' ) + ' ' + method.id
-						}
+						errorMessage={ __( 'There was a problem with the payment method:' ) + ' ' + method.id }
```

Comment:
Should this use (the JS equivalent of) `sprintf`?

Reply:
Oh, good point. I'll change that.